resources/public/api/conf/4.0/self_employment_retrieve.yaml

```diff
@@ -40,67 +63,67 @@ get:
       description: Success
       headers:
         X-CorrelationId:
-          $ref: ./common/headers.yaml#/components/parameters/correlationId
+          $ref: "./common/headers.yaml#/components/parameters/correlationId"
         Deprecation:
-          $ref: './common/headers.yaml#/components/parameters/deprecationHeader'
+          $ref: "./common/headers.yaml#/components/parameters/deprecationHeader"
       content:
         application/json:
           schema:
-            $ref: ./schemas/retrieve_self_employment.json
+            $ref: "./schemas/retrieve_self_employment.json"
           examples:
             example-1:
```

Comment:
This might be out of the scope of this ticket but the naming of the examples i.e. "example-1" isn't very descriptive. Maybe we can change the example name to correspond to the retrieved body i.e. "Full Expenses". Same for examples 2 & 3? What do you think?